T20/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using static System.Console;
+
+class Program {
+
+   // Prints the integral & factorial part of the decimal number given by the user.
+   static void Main () {
+      while (true) {
+         GetResponse ("Enter a decimal number : ", out double f);
+         var txt = f.ToString ();
+         if (txt.Contains ('.')) {
+            var split = txt.Split ('.');
+            var (integral, factorial) = (split[0].Replace (".", ""), split[1]);
+            Print ("Integral Part\t: ", integral);

```

Comment:
```suggestion
            Print ("Integral part\t: ", integral);

```
Should have proper upper and lower cases